core/src/main/java/com/novoda/accessibility/ActionsMenuAccessibilityDelegate.java

```diff
@@ -0,0 +1,39 @@
+package com.novoda.accessibility;
+
+import android.os.Bundle;
+import android.support.v4.view.AccessibilityDelegateCompat;
+import android.support.v4.view.accessibility.AccessibilityNodeInfoCompat;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+
+public class ActionsMenuAccessibilityDelegate extends AccessibilityDelegateCompat {
+
+    private final Menu menu;
+    private final MenuItem.OnMenuItemClickListener menuItemClickListener;
+
+    public ActionsMenuAccessibilityDelegate(Menu menu, MenuItem.OnMenuItemClickListener menuItemClickListener) {
+        this.menu = menu;
+        this.menuItemClickListener = menuItemClickListener;
+    }
+
+    @Override
+    public void onInitializeAccessibilityNodeInfo(View host, AccessibilityNodeInfoCompat info) {
+        super.onInitializeAccessibilityNodeInfo(host, info);
+        for (int i = 0; i < menu.size(); i++) {
+            MenuItem item = menu.getItem(i);
+            info.addAction(new AccessibilityNodeInfoCompat.AccessibilityActionCompat(item.getItemId(), item.getTitle()));
+        }
+    }
+
+    @Override
+    public boolean performAccessibilityAction(View host, int actionId, Bundle args) {
+        MenuItem item = menu.findItem(actionId);
+        if (item == null) {
+            return super.performAccessibilityAction(host, actionId, args);
+        } else {
+            menuItemClickListener.onMenuItemClick(item);
```

Comment:
I dunno if this is legit or if we should be casting to `ActionMenuItem` and calling `invokeAction()`, which calls `onMenuItemClick(item)` anyway.

If we do it this way, then at least, we can pass _any_ menu to this class; we don't need to expose `ActionsMenu`.